package.json

```diff
@@ -50,7 +50,8 @@
     "socket.io-client": "1.4.5",
     "style-loader": "0.12.3",
     "underscore": "^1.8.3",
-    "x-editable": "1.5.1"
+    "x-editable": "1.5.1",
```

Comment:
woowow, that was fast!
mmm, we have it in master
https://github.com/mxcube/mxcube3/blob/master/package.json#L53